src/taxes/components/TaxAppLabel.tsx

```diff
@@ -0,0 +1,72 @@
+import { AppAvatar } from "@dashboard/apps/components/AppAvatar/AppAvatar";
+import { AppUrls } from "@dashboard/apps/urls";
+import { TaxStrategyChoicesQuery } from "@dashboard/graphql";
+import { Box, ExternalLinkIcon, Text } from "@saleor/macaw-ui-next";
+import moment from "moment";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface Props {
+  app: TaxStrategyChoicesQuery["shop"]["availableTaxApps"][number];
+}
+
+export const TaxAppLabel: React.FC<Props> = ({ app }) => {
+  const logo = app.brand?.logo?.default
+    ? { source: app.brand?.logo?.default }
+    : undefined;
```

Comment:
I dont want to introduce any revolution but IMO this coupling is not necessary. This is pure UI level which is now heavily bound to how graphQL looks like. You need name, logo, some date. IMO it much clearer if component decalres these fields:

```
interface Props {
name: string;
logo?: string 
createdAt: string
}
```

Component, when used, will accept in props source that originally comes from graphQL... but it doesn't have to. Its easier, simpler and more testable in general